### DISCUSSION TITLE
Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by benoe77** [jrandiny/yoga-slim7-ubuntu#59] [General]
Did anyone upgraded already to 22.04? What is the experience? Are there any improvements?
Should I wait for the stable version? 
Was reading about kernel level improvements in 5.17 (P-State Driver). Unfortunately 22.04 will come with 5.15 only. Should I change distro? Any experience?

**Comment by arnaud-feldmann** [comment]
Haven't tried 22.04. Though, my Arch in 5.16 works like a charm so the kernel updates improve the behavior. Hope the kernel in Ubuntu LTS works

**Reply by benoe77** [reply]
Thanks, I'm eyeing with Manjaro as a middle(ish)ground between arch and ubuntu.

**Reply by rcapell** [reply]
When you write 'works like a charm', do you mean no kernel patches/S3 advertising tweaks needed anymore? Thanks!

**Reply by arnaud-feldmann** [reply]
No DSDT patch, no dangerous bios tweaks. sleep, hibernate, shutdown are totally stable (i've not had a single pb in 2 weeks of tries).
The only thing that still lacks is performance management. There's still a quiet journal log error at boot about that being unreckonized, but there's no consequence beside not being able to change performance mode without going to the bios.

So yeah there's a big improvement compared to unstable 5.13 with sleep

**Reply by rcapell** [reply]
Thanks @arnaud-feldmann, see also my reply re ubuntu daily.

**Comment by rcapell** [comment]
I installed Ubunty Jammy daily (22.04) now, suspend works with lid close and button press. Kernel `5.15.0-23-generic`. For more details, let me know what you wish to know (and ideally, how to get there:D).

I don't know what suspend state it actually goes into, but from
```
sudo dmesg | grep ACPI:
```
I get
```
...
[    0.383673] ACPI: AC: AC Adapter [ACAD] (off-line)
[    0.383759] ACPI: button: Power Button [PWRB]
[    0.383786] ACPI: button: Sleep Button [SLPB]
[    0.383833] ACPI: button: Lid Switch [LID]
[    0.383897] ACPI: \_SB_.PLTF.C000: Found 3 idle states
[    0.383909] ACPI: FW issue: working around C-state latencies out of order
[    0.383979] ACPI: \_SB_.PLTF.C001: Found 3 idle states
[    0.383983] ACPI: FW issue: working around C-state latencies out of order
[    0.384139] ACPI: \_SB_.PLTF.C002: Found 3 idle states
[    0.384148] ACPI: FW issue: working around C-state latencies out of order
[    0.384313] ACPI: \_SB_.PLTF.C003: Found 3 idle states
[    0.384321] ACPI: FW issue: working around C-state latencies out of order
[    0.384398] ACPI: \_SB_.PLTF.C004: Found 3 idle states
[    0.384403] ACPI: FW issue: working around C-state latencies out of order
[    0.384517] ACPI: \_SB_.PLTF.C005: Found 3 idle states
[    0.384525] ACPI: FW issue: working around C-state latencies out of order
[    0.384648] ACPI: \_SB_.PLTF.C006: Found 3 idle states
[    0.384654] ACPI: FW issue: working around C-state latencies out of order
[    0.384788] ACPI: \_SB_.PLTF.C007: Found 3 idle states
[    0.384794] ACPI: FW issue: working around C-state latencies out of order
[    0.386996] ACPI: thermal: Thermal Zone [TZ00] (53 C)
[    2.483286] ACPI: battery: Slot [BAT1] (battery present)
[    2.686338] ACPI: video: Video Device [VGA] (multi-head: yes  rom: no  post: no)
...
```
And:
```
cat /sys/power/mem_sleep
[s2idle]
```

**Reply by arnaud-feldmann** [reply]
Awesome ! I think I'll stick to Arch for now if it doesn't bring me bugs, but that's a relief to see that finally our config becomes functionnal without any weird trick. I'll definetely try the new Ubuntu LTS

**Comment by benoe77** [comment]
What's the power consumption in these distros?

**Reply by arnaud-feldmann** [reply]
It can't be answered for Arch since Arch you choose what you install then it depends what you put on your distribution. But you can definitely be lightweight if that's your goal. I choose to put the entire gnome suite so basically my arch is close to an Ubuntu

**Comment by arnaud-feldmann** [comment]
Note : Linux 5.17.1 is ok so no breaking changes. Still no performance option though.

**Reply by benoe77** [reply]
Performance option has arrived today to 22.04

**Reply by arnaud-feldmann** [reply]
What kernel ?

**Reply by benoe77** [reply]
5.15.0-25-generic
